File: Python/excluindo.py
import mysql.connector

print('Conectando com o MySQL ...')

try:
    mydb = mysql.connector.connect(
    host="localhost",
    user="root",
    password="root",
    database="aula_bd"
    )
    
    sql = "delete from estado where id = %s"
    estado = (6,)

    cursor = mydb.cursor()
    cursor.execute(sql, estado)
    mydb.commit()
    print("Excluído: ", estado)
    print("Registros afetados: ", cursor.rowcount)
except  BaseException as e:
    print("Erro: ", e)
else:
    mydb.close()

File: Java/Excluindo.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;

public class Excluindo {

    public static void main(String[] args) {
        try {
            System.out.println("Conectando com o MySQL ...");
            Class.forName("com.mysql.jdbc.Driver");
            String url = "jdbc:mysql://localhost:3306/aula_bd";

            Connection mydb = DriverManager.getConnection(url, "root", "root");
            
            String sql = "delete from estado where id = ?";
            PreparedStatement ps = mydb.prepareStatement(sql);
            ps.setInt(1, 6);
            int registros = ps.executeUpdate();
            System.out.println("Registros afetados: " + registros);
        } catch (Exception e) {
            System.out.println("Erro: " + e);
        }
    }
}